examples/integrated_gradients_transformers.ipynb

```diff
@@ -11,9 +11,9 @@
    "cell_type": "markdown",
```

Comment:
<p>entier --&gt; entire</p><p>layer <code>128</code>  --&gt; layer of <code>128</code> </p><p><code>2</code>  units --&gt; <code>2</code>  unit</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/537/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>